controller/api/apis.go

```diff
@@ -3460,3 +3460,29 @@ type RESTSelfApikeyData struct {
 	GlobalPermits []*RESTRolePermission            `json:"global_permissions,omitempty"`
 	DomainPermits map[string][]*RESTRolePermission `json:"domain_permissions,omitempty"` // domain -> permissions
 }
+
```

Comment:
1. Please add "Comment" & "CfgType" fields
2. Should we add a field "PrivateRoot bool `json:"private_root"`" ?
3. There should be different data types for GET & PATCH/POST methods. See RESTAdmissionRule vs. RESTAdmissionRuleConfig as example

Reply:
This has been implemented.